build.sbt

```diff
@@ -1,6 +1,10 @@
 lazy val root = (project in file(".")).
-  aggregate(contrib, mqtt)
+  aggregate(contrib, mqtt, amqp).
+  enablePlugins(GitVersioning)
 
 lazy val contrib = project
 lazy val mqtt = project
 lazy val amqp = project
+
+git.useGitDescribe := true
```

Comment:
are we sure that's a good idea?
Perhaps needs a small guide how to release, because I don't remember how to properly release so it does not end up with weird hash id in version :)